bundles/action/org.openhab.action.pushbullet/src/main/java/org/openhab/action/pushbullet/internal/PushbulletAPIConnector.java

```diff
@@ -126,7 +136,19 @@ private static boolean sendPush(String botName, String recipient, String title,
         push.setTitle(title);
         push.setBody(body);
         push.setType(type);
-        push.setEmail(recipient);
+
+        if (recipient != null) {
+            if (isValidEmail(recipient)) {
+                logger.trace("    Recipient is an email address");
+                push.setEmail(recipient);
+            } else if (isValidChannel(recipient)) {
+                logger.trace("    Recipient is a channel tag");
+                push.setChannel(recipient);
+            } else {
+                logger.warn("Invalid recipient: {}", recipient);
```

Comment:
These two should be DEBUG, not WARN.

Reply:
Shouldn't we let the user know that he specified an invalid recipient? Sending these two in DEBUG will hide this information.

Reply:
I overlooked the distinction between 'not specified' and 'specified, but invalid'.
With the view that this warning is for the latter case, I agree with you.